src/ros/common/unknown-ros.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as vscode from "vscode";
+
+import * as ros from "../ros";
+
+/**
+ * Provides behavior for unknown ROS environment.
+ */
+export class UnknownROS implements ros.ROSApi {
```

Comment:
I think the `UnknownROS` class might not be necessary and we could/should always wait until we have a valid ROS actuator (ROS1 or ROS2)